articles/azure-arc/servers/ssh-arc-overview.md

```diff
@@ -42,10 +56,7 @@ An Azure user with the Owner or Contributor role assigned for a VM doesn't autom
 
 ### Availability
 
-SSH access to Arc-enabled servers is currently supported in all public cloud regions supported by Arc-enabled servers.
-
-> [!NOTE]
-> There currently is no support for non-public clouds.
+SSH access to Arc-enabled servers is currently supported in all public cloud regions supported by Arc-enabled servers and may work on private cloud environments; however functionality on private clouds is not thoroughly tested, if you encounter an error please see [troubleshooting ssh arc](https://learn.microsoft.com/azure/azure-arc/servers/ssh-arc-troubleshoot?tabs=azure-cli).
```

Comment:
I don't think it's good to disclose that certain clouds are not tested. I think that creates more questions than answers and might decrease confidence in this feature. I think it's better if we just revert this to what it used to be ("SSH access to Arc-enabled servers is currently supported in all regions supported by Arc-enabled servers"), and then if someone else has issues in private cloud they can raise an issue.